doc/getting_started/getting_started.rst

```diff
@@ -263,6 +263,17 @@ Make sure to unset the ZEPHYR_SDK_INSTALL_DIR if you don't use the SDK's host
 tools. See `Building without the Zephyr SDK`_ and `Set Up the Development
 Environment`_ for more details.
 
+Using Custom Cmake Toolchains
+=============================
+
+To use a custom toolchain defined in an external cmake file, export the
+following environment variables:
+
+   .. code-block:: console
+
+      export ZEPHYR_TOOLCHAIN_VARIANT=custom
```

Comment:
not a fan of using "custom" as a variant, how about using a name and look for the toolchain file in a list of location first and be able and another location that can be defined for example using 'ZEPHYR_TOOLCHAIN_PATH', i.e. point to a directory rather than a specific file. So:

ZEPHYR_TOOLCHAIN_VARIANT=mytoolchain
ZEPHYR_TOOLCHAIN_PATH=/xx/yy

where mytoolchain.cmake exists under /xx/yy

Reply:
That would be a better approach indeed.

How about aligning with other 'out-of-tree' variables and making it a 'TOOLCHAIN_ROOT' variable. The file included would be in:

TOOLCHAIN_ROOT/cmake/toolchain/ZEPHYR_TOOLCHAIN_VARIANT.cmake

In the future we could have a single OVERLAY_ROOT variable that would set those *_ROOT variables (BOARD, SOC, ...) automatically if the files are found in this OVERLAY_ROOT path.

Reply:
I just updated the PR with the above solution if you want to consider it!